src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -42,7 +43,7 @@ public InstructorFeedbackEditPageData(AccountAttributes account, String sessionT
     public void init(FeedbackSessionAttributes feedbackSession, List<FeedbackQuestionAttributes> questions,
                      Map<String, Boolean> questionHasResponses,
                      List<StudentAttributes> studentList, List<InstructorAttributes> instructorList,
-                     InstructorAttributes instructor) {
+                     InstructorAttributes instructor, String loadInEditMode) {
```

Comment:
@whipermr5 When I change this, line 80 in `InstructorFeedbackEditPageAction` gives an error because im passing a string. This was the string i received from the GET params, would you recommend parsing the GET param to boolean in the action?